src/main/kotlin/icfp2019/core/DistanceEstimate.kt

```diff
@@ -1,3 +1,50 @@
 package icfp2019.core
 
+import icfp2019.analyzers.GraphAnalyzer
+import icfp2019.analyzers.ShortestPathUsingDijkstra
+import icfp2019.model.GameBoard
+import icfp2019.model.GameState
+import icfp2019.model.Node
+import icfp2019.model.Point
+import org.jgrapht.alg.connectivity.ConnectivityInspector
+import org.jgrapht.alg.spanning.PrimMinimumSpanningTree
+import org.jgrapht.graph.AsSubgraph
+
 data class DistanceEstimate(val distance: Int)
+
+object ConservitiveDistanceAnalyzer : Analyzer<(position: Point) -> Pair<Int, Set<Node>>> {
```

Comment:
object ConservativeDistanceAnalyzer : Analyzer<(position: Point) -> Pair<Int, Set<Node>>> {
```suggestion
object ConservativeDistanceAnalyzer : Analyzer<(position: Point) -> Pair<Int, Set<Node>>> {
```

Spelling fix